### DISCUSSION TITLE
Auxiliary App 2.0.13. Installation error

### DISCUSSION THREAD

**Discussion by Mikka27** [aclap-dev/video-downloadhelper#69] [Help and Q&A] [answered]
### Your question

The software asked me to update the Auxiliary App from 2.0.10 to 2.0.13 but I got an error.
I have Windows 10 64 bits and I'm using Edge browser. Anyone else getting the same error?

**Comment by jcfrog** [comment]
Hi,
What error do you get? Do you mean an installation error?

**Reply by Mikka27** [reply, marked as answer]
Hi,

I downloaded latest updates OF Windows; reboot and now I get no error:
thank you:
